services/core/shared/core/compat/sdk_v5/blocks.js

```diff
@@ -353,30 +366,40 @@ const deleteBlock = async (block) => {
 };
 
 const indexGenesisBlock = async () => {
-	logger.info(`Ìndexing genesis block at height ${genesisHeight}`);
-	const [genesisBlock] = await getBlockByHeight(genesisHeight);
-
-	// Index the genesis block transactions first
-	await indexTransactions([genesisBlock]);
-
-	// Index the genesis block accounts next
-	const accountAddressesToIndex = genesisBlock.asset.accounts
-		.filter(account => account.address.length > 16) // Filter out reclaim accounts
-		.map(account => account.address);
-
-	const PAGE_SIZE = 20;
-	const NUM_PAGES = Math.ceil(accountAddressesToIndex.length / PAGE_SIZE);
-	for (let i = 0; i < NUM_PAGES; i++) {
-		// eslint-disable-next-line no-await-in-loop
-		await indexAccountsbyAddress(
-			accountAddressesToIndex.slice(i * PAGE_SIZE, (i + 1) * PAGE_SIZE),
-			true,
-		);
-	}
+	const blocksDB = await getBlocksIndex();
+	const [indexedGenesisBlock] = await blocksDB.find({ height: genesisHeight });
+
+	if (indexedGenesisBlock !== undefined) {
```

Comment:
This looks more clean:
```suggestion
	if (indexedGenesisBlock) {
```